app/views/analytics/_analytics_by_deal_for_venue.erb

```diff
@@ -0,0 +1,152 @@
+                      }
+                  }]
+              },
+
+              series: [
+                  <% loop = 0 %>
+                  <% num_of_active_deals = @deals_for_venue[0].size%>
+                  <% while loop < num_of_active_deals %>
+                  <% redeem_placement = @deals_for_venue[0][loop].size - 1 %>
+                  {
+                      name: '<%= @deals_for_venue[0][loop][0] %>',
+                      data: [
+                          {
+                              name: "<%= @deals_for_venue[0][loop][0] %>",
+                              y: <%= @deals_for_venue[0][loop][redeem_placement] %>,
+                              drilldown: "<%= @deals_for_venue[0][loop][0] %>" + " active"
+                          }
+                      ]
+                  },
+                  <% loop = loop + 1 %>
+                  <% end %>
+              ],
+              drilldown: {
+                  series: [
+                      <% loop = 0 %>
+                      <% while loop < num_of_active_deals %>
+                      {
+                          name: "<%= @deals_for_venue[0][loop][0] %>",
+                          id: "<%= @deals_for_venue[0][loop][0] %>" + " active",
+                          data: [
+                              <% inner_loop = 1 %>
+                              <% num_of_deals = @deals_for_venue[0][loop].size - 1 %>
+                              <% while inner_loop < num_of_deals %>
+                              [
+                                  "<%= @deals_for_venue[0][loop][inner_loop][0] %>", <%= @deals_for_venue[0][loop][inner_loop][1] %>
+                              ],
+                              <% inner_loop = inner_loop + 1 %>
+                              <% end %>
+                          ]
+                      },
+                      <% loop = loop + 1 %>
+                      <% end %>
+
+                      <% loop = 0 %>
+                      <% while loop < num_of_expired_deals %>
+                      {
+                          name: "<%= @deals_for_venue[1][loop][0] %>",
+                          id: "<%= @deals_for_venue[1][loop][0] %>" + " past",
+                          data: [
+                              <% inner_loop = 1 %>
+                              <% num_of_deals = @deals_for_venue[1][loop].size - 1 %>
+                              <% while inner_loop < num_of_deals %>
+                              [
+                                  "<%= @deals_for_venue[1][loop][inner_loop][0] %>", <%= @deals_for_venue[1][loop][inner_loop][1] %>
+                              ],
+                              <% inner_loop = inner_loop + 1 %>
+                              <% end %>
+                          ]
+                      },
+                      <% loop = loop + 1 %>
+                      <% end %>
+
+                  ]
+              }
+          });
+
+          <% loop = 0 %>
```

Comment:
@Woonyong92 Use the highchart scrollable legends for now. By default, toggle off expired deals and show only active deals. But show all past and active deals on label for merchants to toggle on, off.